region/maridia/inner-pink/Colosseum.json

```diff
@@ -700,6 +708,21 @@
       ],
       "note": "Jumping to the first platform is relatively tight without a wall jump and requires disabling HiJump."
     },
+    {
+      "link": [2, 1],
+      "name": "Come In Shinecharged, Shinespark (HiJump Gravity Jump)",
```

Comment:
There's a clip for gravity+walljump, but there isn't a strat for it, on [2. 1]

Reply:
This clip?: https://videos.maprando.com/video/3624

I think there's no reason to include a strat for it, since with Gravity + canConsecutiveWalljump you can already cross the room more easily without a shinespark? Sparking would require canConsecutiveWalljump too.